### ISSUE TITLE
Make the focus style consistent on various buttons on benefits-directory

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#1403]
Currently it's blue, should be yellow. Print, Favourites page, Save for later.

**Comment by smcmurtry** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility work](https://trello.com/c/H7Q1Ycrm/93-accessibility-work)

**Comment by smcmurtry** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility work](https://trello.com/c/H7Q1Ycrm/93-accessibility-work)

**Comment by smcmurtry** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility: Keyboard Navigation](https://trello.com/c/McZemABK/99-accessibility-keyboard-navigation)
